Mjml.Net/Components/Body/ButtonComponent.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mjml.Net.Components.Body
+{
+    public sealed class ButtonComponent : BodyComponentBase
+    {
+        public override string ComponentName => "mj-button";
+
+        public override AllowedAttributes? AllowedAttributes { get; } =
+            new AllowedAttributes
+            {
+                ["name"] = AttributeTypes.String,
```

Comment:
Can we sort these things alphabetically?

Reply:
Sure